### PR TITLE
chore: add dev-dependency disclaimer

### PR DESCRIPTION
**Pull request by mathur07**
SS
image

### CODE REVIEW

File: README.md
Comment (arajkumar) [inline README.md:134]: IMHO, this should be moved to the top, probably after the short intro about the extension.

@prashbnair WDYT?

**Comment by maheshwari-git** [comment]
lgtm

File: README.md
Comment (prashbnair) [reply README.md]: Yes, it should be placed on top.

Comment (mathur07) [reply README.md]: ack

**Comment by mathur07** [comment]
@arajkumar PTAL

File: README.md
Comment (arajkumar) [inline README.md:16]: How about this?
```suggestion
> :warning: Dependency Analytics does not analyse **dev/test** dependencies.
```

Comment (prashbnair) [inline README.md:16]: **Disclaimer**: Dependency Analytics does not analyse **dev/test** dependencies associated with the Stack.

Comment (prashbnair) [inline README.md:16]:
```suggestion
> **Disclaimer**: Dependency Analytics does not analyse **dev/test** dependencies associated with the Stack.
```

Comment (prashbnair) [reply README.md:16]: I feel disclaimer is required. Also dev/test needs to be highlighted.

Comment (mathur07) [reply README.md]: ack

Comment (mathur07) [reply README.md]: ack

**Review by arajkumar — APPROVED** [review]
lgtm, wait for @prashbnair 's approval.

File: README.md
Comment (prashbnair) [inline README.md:16]: Do we need warning and disclaimer, just disclaimer should be good enough.

Comment (mathur07) [reply README.md:16]: okay, let's remove the warning then.

**Pull request merged**
